SECURITY.md

```diff
@@ -1,5 +1,7 @@
 # Security Policy
-Intel is committed to rapidly addressing security vulnerabilities affecting our customers and providing clear guidance on the solution, impact, severity and mitigation. 
+
+Intel is committed to rapidly addressing security vulnerabilities affecting our customers and providing clear guidance on the solution, impact, severity and mitigation.
```

Comment:
Should this be HPE?

Reply:
Probably I was just fixing spelling errors that the Code Spell Checker VSC plug-in was flagging.